helix-core/src/test/resources/log4j.properties

```diff
@@ -27,6 +32,8 @@ log4j.appender.R=org.apache.log4j.RollingFileAppender
 log4j.appender.R.layout=org.apache.log4j.PatternLayout
 log4j.appender.R.layout.ConversionPattern=%5p [%C:%M] (%F:%L) - %m%n
 log4j.appender.R.File=target/ClusterManagerLogs/log.txt
+log4j.appender.R.MaxBackupIndex=200
```

Comment:
I think we discussed, there is no need to configure MaxBackupIndex, right?

Reply:
Let me remove it. 

This thing is that if you really debug some test and enable some extensive logging. You probably still need to enable this one again.

Reply:
removed.